javascript/javascript2/week3/Clicking Game/Clicking Game Thing.js

```diff
@@ -0,0 +1,96 @@
+const btn = document.getElementById("startBtn");
+let inputTime = document.getElementById('inputTime');
+const timer = document.getElementById("timer");
+const scoreS = document.getElementById("scoreS");
+const scoreL = document.getElementById("scoreL");
+const buttonForS = document.getElementById("right");
+const buttonForL = document.getElementById("left");
+
+
+let startNUmS = 0;
+let startNUmL = 0;
+
+btn.addEventListener("click", () => {
+    let StartingSec = inputTime.value;
```

Comment:
Try to make some change in input filed validation, so user can't put negative input value.

Reply:
Also it's a good practice to put a default value for the timer, so if the user doesn't put anything, it doesn't give an error.